.github/dependabot.yml

```diff
@@ -1,127 +1,16 @@
-    versions:
-    - 2.4.3
-    - 2.4.4
-    - 2.4.5
-  - dependency-name: net.serenity-bdd:serenity-gradle-plugin
-    versions:
-    - 2.3.13
-    - 2.4.4
-  - dependency-name: org.springframework:spring-context-support
-    versions:
-    - 5.3.3
-    - 5.3.4
-    - 5.3.6
-  - dependency-name: org.springframework.security:spring-security-test
-    versions:
-    - 5.4.4
-    - 5.4.6
-  - dependency-name: org.springframework.security:spring-security-web
-    versions:
-    - 5.4.6
-  - dependency-name: org.springframework.security:spring-security-crypto
-    versions:
-    - 5.4.6
-  - dependency-name: org.springframework.security:spring-security-core
-    versions:
-    - 5.4.6
-  - dependency-name: org.springframework.security:spring-security-config
-    versions:
-    - 5.4.6
-  - dependency-name: se.patrikerdes.use-latest-versions
-    versions:
-    - 0.2.16
-  - dependency-name: org.springframework:spring-webmvc
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-web
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-tx
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-orm
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-jdbc
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-jcl
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-expression
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-core
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-context
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-beans
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-aspects
-    versions:
-    - 5.3.5
-  - dependency-name: org.springframework:spring-aop
-    versions:
-    - 5.3.5
-  - dependency-name: au.com.dius.pact
-    versions:
-    - 4.1.16
-    - 4.2.0
-    - 4.2.1
-    - 4.2.2
-    - 4.2.3
-    - 4.2.4
-  - dependency-name: org.hibernate:hibernate-core
-    versions:
-    - 5.4.29.Final
-  - dependency-name: info.solidsoft.pitest
-    versions:
-    - 1.6.0
-  - dependency-name: net.javacrumbs.shedlock:shedlock-spring
-    versions:
-    - 4.20.1
-  - dependency-name: net.javacrumbs.shedlock:shedlock-provider-jdbc-template
-    versions:
-    - 4.20.1
-  - dependency-name: com.azure:azure-storage-blob
-    versions:
-    - 12.9.0
-  - dependency-name: org.springframework.cloud:spring-cloud-starter-netflix-hystrix-dashboard
-    versions:
-    - 2.2.6.RELEASE
-    - 2.2.7.RELEASE
-  - dependency-name: org.apache.tomcat:tomcat-annotations-api
-    versions:
-    - 10.0.2
-  - dependency-name: org.apache.tomcat.embed:tomcat-embed-core
-    versions:
-    - 10.0.2
-  - dependency-name: org.junit.jupiter:junit-jupiter-api
-    versions:
-    - 5.7.1
-  - dependency-name: org.junit.jupiter:junit-jupiter-engine
-    versions:
-    - 5.7.1
+    -   package-ecosystem: gradle
```

Comment:
Not sure if this will work as the indentation seems to be out ?

Reply:
I used the IntelliJ shortcut for indentation (command + alt + L), but it seems to have done it weirdly

Reply:
I think it should be correct now